reporting/api/src/main/java/org/jboss/windup/reporting/config/Hint.java

```diff
@@ -187,8 +188,8 @@ public void performParameterized(final GraphRewrite event, final EvaluationConte
             if (locationModel.getFile() instanceof SourceFileModel)
                 ((SourceFileModel) locationModel.getFile()).setGenerateSourceReport(true);
 
-            LOG.info("Hint added to " + locationModel.getFile().getPrettyPathWithinProject() + " [" + this.toString(hintModel.getTitle(), hintText)
-                        + "] with tags: " + StringUtils.join(this.getTags(), " "));
+            LOG.info("Hint added to " + locationModel.getFile().getPrettyPathWithinProject()
+                    + ":\n    " + this.toString(hintModel.getTitle(), null));
```

Comment:
It is better to use `System.lineSeparator()` instead of `\n` to be the same on all OS.

Reply:
Btw how does it matter if our logs are full of newlines coming from our rules?

Reply:
Bump. We have \n all over the project, so unless we are determined to fix all of them, I think there is no need to hold this PR because of that.

Reply:
@OndraZizka - Fixing the PR is trivial. I agree that we should fix some of the other places where we do this, but that isn't a reason to add more places that need fixing. Just take the 5 minutes to fix and repush. :)

Reply:
Ok, but the code is then cluttered with unnecessary calls. The Windows world is now accepting `\n` just ok.